projects/02-dataverse/tests-read-only/oa/oa-prompting.spec.js

```diff
@@ -109,7 +111,7 @@ describe('Test de data set', () => {
         it('Los nombres de todas las propiedades deben tener formato camelCase', () => {
           expect.hasAssertions();
           Object.keys(item.facts).forEach((k) => {
-            expect(/^([a-z]+)(([A-Z]([a-z]+))+)$/.test(k)).toBe(true);
+            expect(/^[a-z]+([A-Z][a-z]*)*$/.test(k)).toBe(true);
```

Comment:
que hace este regex ahora?
si estamos quitando el camelCase requieremiento, deberia cambiar el `it`

Reply:
Acá la regex pasada no estaba funcionando correctamente. Esta nueva propuesta ya verifica si las propiedades son camelCase incluso si hay propiedades de una palabra o más.